### DISCUSSION TITLE
Bell

### DISCUSSION THREAD

**Discussion by opekope2** [opekope2/StaffMod#21] [Staff blocks]
The bell is shown at [13:57](https://youtu.be/vlVUVsx6BYY?t=837). It rings, when it hits an entity.

Attack damage: more than an empty staff (?)
Attack speed: less than an empty staff (?)

When attacking, the bell rings
Other mechanics to discuss:
* Work like a bell in the village: alert villagers to go back to their houses
* Make hostile mobs glow (the same ones as a [vanilla bell does](https://minecraft.wiki/w/Bell#Glowing_effect))

**Comment by TheDogeGuy** [comment]
ohhhhhhhhh i thought the bell was just useless but i might use it in raids

**Reply by opekope2** [reply]
It is "useless" for now, but I plan to implement those "other mechanics" in a future release.
